src/neuroconv/datainterfaces/behavior/fictrac/fictracdatainterface.py

```diff
@@ -209,6 +214,8 @@ def add_to_nwbfile(
             write_timestamps = True
 
         position_container = Position(name="FicTrac")
+        if self.configuration_metadata is not None:
+            comments = f"configuration_metadata = {json.dumps(self.configuration_metadata)}"
```

Comment:
My only problem with this kind of embedding is if someone wanted to reload the JSON object they would have to restrip the extra bit of string at the start

Reply:
Yeah, that's true. The advantage of this is that is a bit more specific about what is the meaning of this field. At least more specific that receving just a json dictionary with no context. 

Plus, I don't think that realoding the json object with everything as a string is that useful without the parsing function.